src/Sulu/Bundle/RouteBundle/Command/UpdateRouteCommand.php

```diff
@@ -32,6 +32,7 @@ protected function configure()
     {
         $this->setName('sulu:route:update')
             ->addArgument('entity', InputArgument::REQUIRED)
+            ->addArgument('locale', InputArgument::REQUIRED)
```

Comment:
The localizations have to be configured somewhere, shouldn't they? Can't we just use the `LocalizationProvider`?